sycl/unittests/Extensions/WorkGroupMemoryBackendArgument.cpp

```diff
@@ -0,0 +1,41 @@
+//==--------------------- WorkGroupMemoryBackendArgument.cpp --------------===//
```

Comment:
Although I've explicitly checked using FileCheck that only one call to urKernelSetArgLocal is made, this unit test shows up as failure for me for some reason. I suspect I'm not using the unit test infrastructure properly.